app/static/js/initialize_map.js

```diff
@@ -0,0 +1,119 @@
+geocoder = new google.maps.Geocoder();
```

Comment:
Let's put this file in assets instead of static - that way it will be [minified](https://en.wikipedia.org/wiki/Minification_(programming)) and compressed